android/app/src/main/java/app/organicmaps/sdk/maplayer/subway/SubwayManager.java

```diff
@@ -1,20 +1,18 @@
-package app.organicmaps.maplayer.subway;
+package app.organicmaps.sdk.maplayer.subway;
 
-import android.app.Application;
 import android.content.Context;
 
 import androidx.annotation.NonNull;
 
-import app.organicmaps.Framework;
-import app.organicmaps.MwmApplication;
+import app.organicmaps.sdk.Framework;
 
 public class SubwayManager
 {
   @NonNull
   private final OnTransitSchemeChangedListener mSchemeChangedListener;
 
-  public SubwayManager(@NonNull Application application) {
-    mSchemeChangedListener = new OnTransitSchemeChangedListener.Default(application);
+  public SubwayManager(@NonNull Context context) {
```

Comment:
Is it possible to check/fix code style here and in other modified places automatically?
```suggestion
  public SubwayManager(@NonNull Context context)
  {
```